Tools-Override/tests.targets

```diff
@@ -245,6 +239,7 @@
           >
 
     <MakeDir Condition="'$(CoverageEnabledForProject)'=='true'" Directories="$(CoverageReportDir)" />
+    <Error Text="TestILCFolder property is required for running uapaot tests. Please pass in the full path to where ilc.exe to msbuild using /p:TestILCFolder=path_to_ilc.exe" Condition="'$(BuildingUAPAOTVertical)' == 'true' AND '$(TestILCFolder)' == ''" />
```

Comment:
This is simply a friendly message that will be displayed when people try to run tests for uapaot and don't pass in the TestILCFolder path